### PR TITLE
Support for azure storage and OpenStack Swift

### PR DESCRIPTION
**Pull request by iamrz1**

### CODE REVIEW

**Comment by tamalsaha** [comment]
Configure Azure folder
https://github.com/kubedb/wal-g/blob/release-0.2.4-ac/internal/configure.go#L177

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 CMD_FILES = $(wildcard cmd/wal-g/*.go)
 PKG_FILES = $(wildcard internal/**/*.go internal/**/**/*.go internal/*.go)
 TEST_FILES = $(wildcard test/*.go testtools/*.go)
+PKG := github.com/wal-g/wal-g
```
Comment (the-redback) [inline Makefile]: Remove alpine build changes. Because that will be a different PR in wal-g. 

xref: https://github.com/kubedb/wal-g/commit/14ba013e6ce065e649b3818a1dcd6141ae8b3f37

File: internal/config.go [hunk 2]
```diff
@@ -103,4 +107,4 @@ func getSettingValue(key string) string {
 		return value
 	}
 	return ""
-}
+}
```
Comment (the-redback) [inline internal/config.go]: New line at the end.

**Comment by tamalsaha** [comment]
LGTM. @iamrz1 , please open a pr against the wal-g/wal-g repo.
